permissions/Accounts.permission

```diff
@@ -21,6 +21,10 @@ whitelist ${HOME}/.local/share/accounts
 dbus-user.talk com.google.code.AccountsSSO.SingleSignOn
 dbus-user.broadcast com.google.code.AccountsSSO.SingleSignOn=*@/*
 
+# libaccounts-glib/libaccounts-qt
+dbus-user.own com.google.code.AccountsSSO.Accounts.*
```

Comment:
Is this needed? If this is just about being able to listen these com.google.code.AccountsSSO.Accounts.uuid_* names then, this shouldn't be needed as this is about owning those names (i.e. reserving a bus name for the sandboxed application itself).

Reply:
Yeah, the library is both a signal listener and an emitter, so it needs this to be able to own a name to send the signals out on (at least that's what I understood from our discussion: it doesn't need the name to send the signals, but it does need it to allow the signals to be filtered).

All of the instances basically store their data in the same central database, but they signal to each other that they've made a change over dbus in an attempt to keep all the instances synchronised. It's all done this way because there's no central daemon with a single view of the world.